### PR TITLE
Add jacoco test and action

### PR DESCRIPTION
**Pull request by matthewgardner**
Add Jacoco to build
Add publish jacoco build to PR action

https://github.com/finos-labs/architecture-as-code/issues/189

### CODE REVIEW

**Review by rocketstack-matt — APPROVED** [review]

**Comment by matthewgardner** [comment]
Do not merge please - for reasons Im trying to work out - its not finding the file :(

**Comment by matthewgardner** [comment]
something to come back to - i need to do some more research cos all the googling on how to solve this hasnt work :(

Ready to merge - unless you want the disabled action code removed? @rocketstack-matt @Budlee

**Review by rocketstack-matt — APPROVED** [review]

**Pull request merged**
